include/aws/cryptosdk/materials.h

```diff
@@ -400,7 +400,14 @@ struct aws_cryptosdk_cmm_vt {
 /**
  * Putting this here for now, until we get it merged into the atomics.h in c-common
  */
-AWS_CRYPTOSDK_STATIC_INLINE bool aws_atomic_var_is_valid(const struct aws_atomic_var *var) {
+AWS_CRYPTOSDK_STATIC_INLINE bool aws_atomic_var_is_valid_int(const struct aws_atomic_var *var) {
+    return AWS_MEM_IS_WRITABLE(var, sizeof(size_t));
+}
+
+/**
+ * Putting this here for now, until we get it merged into the atomics.h in c-common
+ */
```

Comment:
Do we have a GitHub issue to track this? It would be nice to get @nchong-at-aws input here too.

Reply:
Yep, we do! https://github.com/aws/aws-encryption-sdk-c/issues/612